source/beamline/VESPERS/VESPERSBeamline.cpp

```diff
@@ -800,6 +800,15 @@ void VESPERSBeamline::setupExposedControls()
 	addExposedControl(pseudoWireStageMotorGroupObject()->normalControl());
 	addExposedControl(bigBeamMotorGroupObject()->horizontalControl());
 	addExposedControl(bigBeamMotorGroupObject()->verticalControl());
+	addExposedControl(pseudoAttocubeStageMotorGroupObject()->horizontalControl());
+	addExposedControl(pseudoAttocubeStageMotorGroupObject()->verticalControl());
+	addExposedControl(pseudoAttocubeStageMotorGroupObject()->normalControl());
+	addExposedControl(realAttocubeStageMotorGroupObject()->horizontalControl());
+	addExposedControl(realAttocubeStageMotorGroupObject()->verticalControl());
+	addExposedControl(realAttocubeStageMotorGroupObject()->normalControl());
+	addExposedControl(attocubeRxMotorGroupObject()->horizontalControl());
```

Comment:
They actually are.  The labels of horizontal, vertical, and normal don't translate well to rotation.  It's a bit of a limitation of the original design.